contracts/reverseRegistrar/IL2ReverseRegistrar.sol

```diff
@@ -0,0 +1,24 @@
+pragma solidity >=0.8.4;
+
+interface IL2ReverseRegistrar {
+    function setName(string memory name) external returns (bytes32);
+
+    function setNameForAddrWithSignature(
+        address addr,
+        string memory name,
+        address relayer,
```

Comment:
Why does `relayer` need to be specified here?

Reply:
I was thinking if someone wanted to be more restrictive on who could send the tx, but maybe it doesn't matter given that all of the arguments are set within the signature.